src/routes/[lang=lang]/+page.svelte

```diff
@@ -7,18 +7,19 @@
 	let spectators = 0
 
 	onMount(() => {
-		const interval = setInterval(updateSpectatorCount, 2_000)
+		const interval = setInterval(async () => {
```

Comment:
I would write it like this:
```ts
const interval = setInterval(updateSpectatorCount, 2_000)
```
The `updateSpectatorCount` is a bit misleading since the function does not really uodate the count.